tox.ini

```
@@ -1,13 +1,13 @@
 [tox]
-envlist = isort, black, flake8, safety, docs, py27, py3{5,6,7,8}-symengine
+envlist = isort, black, flake8, safety, docs, py27, py3{6,7,8}-symengine, py39
```

It doesn't really matter much, since the `gh-actions` section lists specifically what should be tested, but why not list the envs as follows?

```suggestion
envlist = isort, black, flake8, safety, docs, py27, py3{6,7,8,9}-symengine
```

---

There are no wheels for symengine for Python 3.9 for all platforms. So this would fail for windows in the installation step. Also I noticed that if it's not set here it does not get run on actions.

---

Okay, in that case it should probably be removed below.